AdtSampleApp/SampleFunctionsApp/AdtUtilities.cs

```diff
@@ -32,12 +29,13 @@ await foreach (IncomingRelationship ie in rels)
             }
             return null;
         }
-        public static async Task<string> FindParentByQuery(DigitalTwinsClient client, string childId, ILogger log)
+
+        public static async Task<string> FindParentByQueryAsync(DigitalTwinsClient client, string childId, ILogger log)
         {
-            string query = $"SELECT Parent " +
-                            $"FROM digitaltwins Parent " +
-                            $"JOIN Child RELATED Parent.contains " +
-                            $"WHERE Child.$dtId = '" + childId + "'";
+            string query = "SELECT Parent " +
+                           "FROM digitaltwins Parent " +
```

Comment:
Would this be prettier using $ notation:
"SELECT Parent " +
"FROM digitaltwins Parent " +
"JOIN Child RELATED Parent.contains " +
$"WHERE Child.$dtId = '{childId}'";

Reply:
I'd prefer it that way too.